### PR TITLE
Create Modular Exercise List Application

### LINKED ISSUE #22: Create Exercise Commands Parser
Example commands:
```
Exercise: exercise <command> <arguments>
list: exercise list
create: exercise create e/<exercise>
update: exercise edit e/<old_exercise_name> f/<new_exercise_name>
delete: exercise delete e/<exercise>

```

### LINKED ISSUE #24: Add CRUD to Exercise
```
list: exercise list
create: exercise create e/<exercise>
update: exercise edit e/<old_exercise_name> f/<new_exercise_name>
delete: exercise delete e/<exercise>
```

### LINKED ISSUE #52: Create Exercise Parser Tests
Objective: Be able to test parsing these commands:
```
list: exercise list
create: exercise create e/<exercise>
update: exercise edit e/<old_exercise_name> f/<new_exercise_name>
delete: exercise delete e/<exercise>
```

### LINKED ISSUE #65: Aloysius: Exercise

### PR DESCRIPTION
**Pull request by alloystory**
1. Removed all references to AddressBook-Level-3
2. Added classes for Exercise and ExerciseList. These templates can be used by developers to create the other features of the application.
3. Modularised Application. This allows developers to easily create new features for the application
4. Changed test cases to fit the current Exercise List application.

Resolved #52 #65 #24 #22

### CODE REVIEW

File: src/main/resources/view/ExerciseListCard.fxml [hunk 1]
```diff
@@ -3,34 +3,34 @@
...
-        <Label fx:id="id" styleClass="cell_big_label">
+      <HBox alignment="CENTER_LEFT" spacing="5">
+        <Label fx:id="exerciseId" styleClass="cell_big_label" text="1. ">
           <minWidth>
             <!-- Ensures that the label text is never truncated -->
             <Region fx:constant="USE_PREF_SIZE" />
           </minWidth>
         </Label>
-        <Label fx:id="name" text="\$first" styleClass="cell_big_label" />
+        <Label fx:id="exerciseName" styleClass="cell_big_label" text="Bench Press" />
```
Comment (gb3h) [inline src/main/resources/view/ExerciseListCard.fxml:28]: Is the text for testing purposes?

File: src/test/data/exercise/JacksonExerciseListTest/typicalExercisesExerciseList.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+    "_comment": "ExerciseList save file which contains the same Person values as in TypicalExercises#getTypicalExerciseList()",
```
Comment (gb3h) [inline src/test/data/exercise/JacksonExerciseListTest/typicalExercisesExerciseList.json]: I think Person should be Exercise

**Review by gb3h — APPROVED** [review]
LGTM!

File: src/main/resources/view/ExerciseListCard.fxml (hunk 1 above)
Comment (alloystory) [reply src/main/resources/view/ExerciseListCard.fxml:28]: yeap. the text is for you to see a sample in SceneBuilder. this text will be replaced in the actual application.

File: src/test/data/exercise/JacksonExerciseListTest/typicalExercisesExerciseList.json (hunk 2 above)
Comment (alloystory) [reply src/test/data/exercise/JacksonExerciseListTest/typicalExercisesExerciseList.json]: ah yes. sorry about that

**Review by Guofeng-Tang — APPROVED** [review]
LGTM!

**Pull request merged**